### PR TITLE
Release smart_proxy_ansible 1.0.1 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [x] 1.13
* [x] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Relies on https://github.com/theforeman/foreman-packaging/pull/1497  to work - thanks! Please merge for all possible versions

### CODE REVIEW

**Comment by mmoll** [comment]
Please push smart_proxy_ansible 1.0.1 to rubygems.

**Comment by dLobatog** [comment]
@mmoll Pushed, I was granted rights a few minutes ago only, my bad 😄  https://github.com/theforeman/foreman-packaging/pull/1499

[test]

File: plugins/smart_proxy_ansible/ansible.rb [hunk 1]
```diff
@@ -1,2 +1,2 @@
-gem 'smart_proxy_ansible', '0.0.1'
-gem 'foreman_ansible_core'
+gem 'smart_proxy_ansible', '1.0.1'
+gem 'foreman_ansible_core', '~> 1.0.0'
```
Comment (mmoll) [inline plugins/smart_proxy_ansible/ansible.rb]: that version dependencies should also get reflected in the control file

**Comment by dLobatog** [comment]
@mmoll Updated to reflect the version dependency on the control file

**Comment by mmoll** [comment]
Shouldn't `~> 1.0.0` translate to `(>= 1.0.0)` and `(<< 1.1.0)`? Also, is that strict pessimistic constraint really required?

**Comment by dLobatog** [comment]
@mmoll It isn't really required, I overlooked it and pushed the gem - anyway the latest version available of foreman_ansible_core is 1.0.0. I will make it less strict (~> 1.0) on the next release.

I've updated the control file to reflect < 1.1.0  which is what https://github.com/theforeman/smart_proxy_ansible/blob/master/bundler.plugins.d/foreman_ansible_core.rb says

**Comment by dLobatog** [comment]
Replaced by https://github.com/theforeman/foreman-packaging/pull/1526

**Pull request closed without merging**